EnglishFilter/sections/antiadblock.txt

```diff
@@ -574,6 +574,9 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80254
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,domain=vladan.fr
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=vladan.fr
```

Comment:
missed `@@`? So redirect `||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=vladan.fr`

Reply:
Yep, I missed. How silly of me!